docs/source/support/release-notes.rst

```diff
@@ -17,194 +17,191 @@ HEAD — ongoing
-- Java Bindings: ``CreateAndExerciseCommand`` is now properly converted in the Java Bindings data layer (see `#979 <https://github.com/digital-asset/daml/pull/979>`_)
-- Alpha release of :doc:`DAML Integration Kit </daml-integration-kit/index>`.
+- **Contract keys**: Fixed two issues related to contract key visibility.
 
-- Add ``quickstart-scala`` DAML Assistant project template.
+  See `issue #969 <https://github.com/digital-asset/daml/pull/969>`__ and `issue #973 <https://github.com/digital-asset/daml/pull/973>`__ for details.
+- **Java Codegen**: Variants with unserializable cases are now accepted.
 
-- DAML-LF Engine: if all labels in a record are set, fields no longer need to be ordered (see `#988 <https://github.com/digital-asset/daml/issues/988>`_)
+  See `issue #946 <https://github.com/digital-asset/daml/pull/946>`__ for details.
+- **Java Bindings**: ``CreateAndExerciseCommand`` is now properly converted in the Java Bindings data layer.
+
+  See `issue #979 <https://github.com/digital-asset/daml/pull/979>`__ for details.
+- **DAML Integration Kit**: Alpha release of the kit for integrating your own ledger with DAML. See the :doc:`DAML Integration Kit docs </daml-integration-kit/index>` for how to try it out.
+- **DAML Assistant**: Added a ``quickstart-scala`` DAML Assistant project template.
+- **DAML-LF Engine**: If all labels in a record are set, fields no longer need to be ordered.
+
+  See `issue #988 <https://github.com/digital-asset/daml/issues/988>`__ for details.
 
 .. _release-0-12-15:
 
 0.12.15 - 2019-05-06
 --------------------
 
-- Beta release of the Windows SDK:
-  You can download the installer from
-  `GitHub releases <https://github.com/digital-asset/daml/releases>`_.
-  The Windows SDK ships with the new `daml` installer which will soon also
-  become the default on Linux and MacOS. Documentation is still in progress,
-  take a look at the `Migration guide <https://github.com/digital-asset/daml/pull/768>`_
-  and the `updated documentation <https://github.com/digital-asset/daml/pull/740>`_.
-- Add ``fromListWith`` and ``merge`` to ``DA.TextMap``.
-- Add ``DA.Next.Map`` and ``DA.Next.Set`` and deprecate ``DA.Map`` and ``DA.Set`` in favor of those.
-- Ledger API: Added three new methods to :ref:`CommandService <com.digitalasset.ledger.api.v1.commandservice>`:
-
-  - ``SubmitAndWaitForTransactionId`` returns the transaction id.
+- **Windows support**: Beta release of the Windows SDK.
+
+  To try it out, download the installer from `GitHub releases <https://github.com/digital-asset/daml/releases>`_. The Windows SDK uses the new ``daml`` command-line which will soon also
+  become the default on Linux and MacOS.
+
+  Documentation is still in progress, but you can see the :doc:`Migration guide </support/new-assistant>` and the `pull request for the updated documentation <https://github.com/digital-asset/daml/pull/740>`__.
+- **DAML**: Added ``fromListWith`` and ``merge`` to ``DA.TextMap``.
```

Comment:
These could be `**DAML Stdlib**` or `**DAML Standard Library**`. Similarly below.

Reply:
sure, will change

Reply:
done